graphql-server/pkg/queries/proposal.go

```diff
@@ -76,6 +78,12 @@ func (q *ProposalQuery) NewQuery() *bun.SelectQuery {
 		})
 	}
 
+	if q.scopedOrder != nil {
+		if q.scopedOrder.SubmitTime != nil {
+			query = query.Order(fmt.Sprintf("submit_time %s", q.scopedOrder.SubmitTime))
```

Comment:
i have bad feeling about this (plugging request values into sql directly); currently graphql handler will validate the order string is `ASC` or `DESC`, so sql injection is not possible in theory...

cc @hochiw